mods/cnc/languages/chrome/en.ftl

```diff
@@ -233,96 +233,93 @@ supportpowers-support-powers-palette =
+     - Vehicles attempt to crush enemies at the target location.
+     - Helicopters land at the target location.
 
-    Left-click icon then right-click on target.
-    Hold <(Alt)> to activate temporarily while commanding units.
+    Left-click the icon then right-click on the target.
+    Hold <(Alt)> to temporarily activate while commanding units.
 
 button-command-bar-force-attack =
    .tooltip = Force Attack
-   .tooltipdesc = Selected units will attack the targeted unit or location
-     - Default activity for the target is suppressed
-     - Allows targeting of own or ally forces
-     - Long-range artillery units will always target the
-       location, ignoring units and buildings
+   .tooltipdesc = Selected units attack the targeted unit or location.
+     - Overrides default activity.
+     - Allows targeting of own or allied forces.
+     - Long-range artillery units always target the
+       location, ignoring units and buildings.
 
-    Left-click icon then right-click on target.
-    Hold <(Ctrl)> to activate temporarily while commanding units.
+    Left-click the icon then right-click on the target.
+    Hold <(Ctrl)> to temporarily activate while commanding units.
 
 button-command-bar-guard =
    .tooltip = Guard
-   .tooltipdesc = Selected units will follow the targeted unit.
+   .tooltipdesc = Selected units follow the targeted unit.
 
-    Left-click icon then right-click on target unit.
+    Left-click the icon then right-click on the target.
 
 button-command-bar-deploy =
    .tooltip = Deploy
-   .tooltipdesc = Selected units will perform their default deploy activity
-     - MCVs will unpack into a Construction Yard
-     - Construction Yards will re-pack into a MCV
-     - Transports will unload their passengers
+   .tooltipdesc = Selected units perform their default deploy activity.
+     - MCVs deploy into Construction Yards.
+     - Construction Yards repack into MCVs.
+     - Transports unload their passengers.
 
     Acts immediately on selected units.
 
 button-command-bar-scatter =
    .tooltip = Scatter
-   .tooltipdesc = Selected units will stop their current activity
-    and move to a nearby location.
+   .tooltipdesc = Selected units stop their current activity
+    and move to nearby cells.
 
     Acts immediately on selected units.
 
 button-command-bar-stop =
    .tooltip = Stop
-   .tooltipdesc = Selected units will stop their current activity.
-    Selected buildings will reset their rally point.
+   .tooltipdesc = Selected units stop their current activity.
+    Selected buildings reset their rally point.
 
     Acts immediately on selected targets.
 
 button-command-bar-queue-orders =
    .tooltip = Waypoint Mode
-   .tooltipdesc = Use Waypoint Mode to give multiple linking commands
-    to the selected units. Units will execute the commands
-    immediately upon receiving them.
+   .tooltipdesc = Issues a series of linked commands to selected units, executed in sequence.
```

Comment:
this rewording is wrong. Try using waypoint mode from command bar to understand it